app/src/main/java/com/android/bookswap/ui/profile/EditProfile.kt

```diff
@@ -0,0 +1,137 @@
+package com.android.bookswap.ui.profile
+
+import android.util.Log
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.android.bookswap.data.DataUser
+import com.android.bookswap.ui.theme.BookSwapAppTheme
+
+@Composable
+fun EditProfileDialog(
+    onDismiss: () -> Unit,
+    onSave: (DataUser) -> Unit,
+    dataUser: DataUser,
+    modifier: Modifier = Modifier
```

Comment:
The parameter modifier is never used, if it isn't supposed to be used, then it can be removed.

Reply:
The modifier is used when this composable is instantiated as part of another component, just like the modifiers from basic Material3 components